### PR TITLE
improving readme on dynamic inventory for ec2.ini options

### PR DESCRIPTION
**Pull request by thapakazi**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
-  docsite/rst/intro_dynamic_inventory wiki-guidelines

##### ANSIBLE VERSION
```
ansible 2.1.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides```
```

##### SUMMARY
This PR is to address the old recurring unfriendly error message like this: https://github.com/ansible/ansible/issues/10840
From the discussion there, seems its better if the issues is address via proper documentation. So writing some guidelines with   _been there done that_ feeling ;) 

Basically the docs explains in more verbose how to properly configure the aws dynamic inventory script settings to avoid unnecessary headache with and example use-case that's common to newbie users fiddling with the ansible-aws-dynamic-host workflow.

@jctanner here it comes my PR 😎

### CODE REVIEW

**Comment by dharmabumstead** [comment]
Merged with edits. Thanks @zerOnepal!

**Comment by thapakazi** [comment]
Thanks for the edits @dharmabumstead :bowing_man: 
...such a privilege and learning experience from awesome people :smile: 
happy_minion
